test/src/test/java/org/corfudb/runtime/view/stream/StreamAddressSpaceTest.java

```diff
@@ -148,7 +148,6 @@ public void testToArray() {
 
     @Test
     public void testEquality() {
-        assertThat(new StreamAddressSpace().equals(null)).isFalse();
```

Comment:
Please bring this back as it's a test for the first condition of the equals method in `StreamAddressSpace`https://github.com/CorfuDB/CorfuDB/pull/2892#discussion_r592750198

Reply:
Done, Thank you very much.